app/views/feed/infinite_scroll_feed_items.js.erb

```diff
@@ -0,0 +1,32 @@
+(() => {
+  <% if @revisions.any? %>
```

Comment:
Even though we're not dealing with Elasticsearch on top of this particular infinite scroll, I still think some error handling similar to [#146 - app/views/posts/infinite_scroll_posts.js.erb](https://github.com/EloHellEsports/workshop.codes/pull/146/files#diff-2c72e52b1f5ccc4382e36da6c172e75715ef23ecf05330f2baddd39ad83e9e64) might be nice just in case.